### PR TITLE
Fix wheel tests

### LINKED ISSUE #2045: Fix wheel testing
In order to test the right wheel I made some changes in #2030. These seem not to work on CI and need to be fixed for 0.4.0

### PR DESCRIPTION
**Pull request by benjeffery**
Fixes #2045  Thanks @grahamgower for the fix.

### CODE REVIEW

**Comment by benjeffery** [comment]
Gah, Looks like it is still not happy.

**Comment by grahamgower** [comment]
Might be time to do something drastic and try to ssh into a running workflow: https://github.com/mxschmitt/action-tmate

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -154,7 +154,7 @@ jobs:
         run: |
           python -VV
           # Install the local wheel
-          pip install dist/tskit-*.whl
```
Comment (jeromekelleher) [inline .github/workflows/wheels.yml:157]: try ``python -m pip install`` - looks to me like it's a mixing-python-versions issue.

**Comment by benjeffery** [comment]
> Might be time to do something drastic and try to ssh into a running workflow: https://github.com/mxschmitt/action-tmate

I know what the issue is, it's just not an easy fix. We have several linux wheels in one archive, and it needs to be told which one to install.

**Comment by benjeffery** [comment]
OK! Think this does the trick.

**Review by jeromekelleher — COMMENTED** [review]
LGTM!

**Review by jeromekelleher — APPROVED** [review]
LGTM!

**Pull request merged**